src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
  * The `get()` method of the `AcceptHeader` class now takes into account the
    `*` and `*/*` default values (if they are present in the Accept HTTP header)
    when looking for items.
+ * Adds `getAcceptableFormats()` for reading acceptable formats based on Accept header
```

Comment:
Should be `added` to be consistent with the wording of other entries in that file.